src/library_webgpu.js

```diff
@@ -1711,8 +1711,8 @@ var LibraryWebGPU = {
         return undefined;
       }
 
-      var depthSlice = {{{ makeGetValue('caPtr', C_STRUCTS.WGPURenderPassColorAttachment.depthSlice, 'i32') }}};
-      {{{ gpu.convertSentinelToUndefined('depthSlice') }}}
+      var depthSlice = {{{ makeGetValue('caPtr', C_STRUCTS.WGPURenderPassColorAttachment.depthSlice, 'u32') }}};
+      if (depthSlice == 0xFFFFFFFF) depthSlice = undefined;
```

Comment:
This looks unrelated to the title+description of the PR?

Reply:
Oops, reverted.